Website/js/Debug/dnn.modalpopup.js

```diff
@@ -201,29 +201,41 @@
                     var $dnnToggleMax = $('<a href="#" class="dnnToggleMax"><span>Max</span></a>');
                     $modal.parent().find('.ui-dialog-titlebar-close').before($dnnToggleMax);
 
-                    $dnnToggleMax.click(function(e) {
+                    var initialTop = $modal.parent().css("top");
+                    var initialLeft = $modal.parent().css("left");
+                    
+                    $dnnToggleMax.click(function (e) {
                         e.preventDefault();
 
                         var $window = $(window),
                             newHeight,
-                            newWidth;
+                            newWidth,
+                            newTop,
+                            newLeft;
 
                         if ($modal.data('isMaximized')) {
-                            newHeight = $modal.data('height');
+                            newHeight = $modal.data('height') + 100;
```

Comment:
That's very weird. We should investigate a bit why we need to sum 100 px in this scenario. In case we cannot fix this using the jQuery UI API I would at least store the 100 pixel in a variable and put a comment explaining this value. i.e.: var JQUERY_UI_HEIGHT_SRINK_OFFSET. Leaving this value without a named variable make the code very hard to understand.